corehq/apps/export/static/export/js/download_data_files.js

```diff
@@ -12,7 +15,10 @@ hqDefine("export/js/download_data_files", function() {
      * @param url: The URL to be copied to the clipboard
      * @param textareaElem: A hidden <textarea> element containing the URL
      */
-    var copyDataFileUrl = function (url, textareaElem) {
+    var copyDataFileUrl = function () {
```

Comment:
Suggestion: this would be a good place to use [assert_properties](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/hqwebapp/static/hqwebapp/js/assert_properties.js) to verify the data attributes you expect are present, something like `assertProperties.assertRequired($(this).data(), ['name', 'url'])`

Reply:
nice one cool. thanx